6 add docs (#8)
mozilla-ai/structured-q-a: README.md

<p align="center"><img src="./images/Blueprints-logo.png" width="35%" alt="Project logo"/></p>

# Structured-Q&A: a Blueprint by Mozilla.ai for answering questions about structured documents.

[![](https://dcbadge.limes.pink/api/server/YuMNeuKStr?style=flat)](https://discord.gg/YuMNeuKStr)
[![Docs](https://github.com/mozilla-ai/structured-q-a/actions/workflows/docs.yaml/badge.svg)](https://github.com/mozilla-ai/structured-q-a/actions/workflows/docs.yaml/)
[![Tests](https://github.com/mozilla-ai/structured-q-a/actions/workflows/tests.yaml/badge.svg)](https://github.com/mozilla-ai/structured-q-a/actions/workflows/tests.yaml/)
[![Ruff](https://github.com/mozilla-ai/structured-q-a/actions/workflows/lint.yaml/badge.svg?label=Ruff)](https://github.com/mozilla-ai/structured-q-a/actions/workflows/lint.yaml/)

> This blueprint demonstrate how you can answer questions about structured documents using open-source models and a simple LLM workflow.

Reviewer: Change to "This Blueprint demonstrates how to use open-source models and a simple LLM workflow to answer questions based on structured documents"

> This blueprint demonstrate how you can answer questions about structured documents using open-source models and a simple LLM workflow.

Author: ```suggestion
This Blueprint demonstrates how to use open-source models and a simple LLM workflow to answer questions based on structured documents.
```

> It is designed to showcase a simpler alternative to more complex and/or resource demanding alternatives, like RAG systems and/or long-context models.

Reviewer: Minor change, i would probably change to: "It is designed to showcase a simpler alternative to more complex and/or resource demanding alternatives, such as RAG systems that rely on vectorDBs and/or long-context models with large token windows" 

I think you could argue that our solution is kinda-RAG?WDYT?

> It is designed to showcase a simpler alternative to more complex and/or resource demanding alternatives, like RAG systems and/or long-context models.

Author: ```suggestion
It is designed to showcase a simpler alternative to more complex and/or resource demanding alternatives, such as RAG systems that rely on vectorDBs and/or long-context models with large token windows.
```
